tools/swig/numpy.i

```diff
@@ -80,6 +80,7 @@
 %#define array_data(a)          (((PyArrayObject*)a)->data)
 %#define array_descr(a)         (((PyArrayObject*)a)->descr)
 %#define array_flags(a)         (((PyArrayObject*)a)->flags)
+%#define array_clearflags(a,f)  (((PyArrayObject*)a)->flags) &= ~f
 %#define array_enableflags(a,f) (((PyArrayObject*)a)->flags) = f
```

Comment:
This isn't yours, but it looks incorrect. Should probably be `|= f`, which is consistent for the later numpy API versions.

Reply:
The line just below mine? They assigned instead of doing the OR. Maybe that's why they never had the issue because the would clear all flags when setting F_CONTIGUOUS. I didn't change it because I'm not setup to test it.

Reply:
Ha, that is a definite possibility, although the version 7 api is pretty old.

Reply:
Maybe a rare case of two wrongs make a right?